src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -1785,6 +1816,24 @@ public function setIgnoreMandatoryFlag($ignoreMandatoryFlag)
     }
 
     /**
+     * Orders a child before another child
+     * @param NodeInterface $parent
+     * @param NodeInterface $childSource
+     * @param NodeInterface $childDest
+     */
+    private function executeOrderBefore(
+        NodeInterface $parent,
+        NodeInterface $childSource,
+        NodeInterface $childDest
+    ) {
+        // reorder
+        $parent->orderBefore($childSource->getName(), $childDest->getName());
+
+        $event = new ContentOrderBeforeEvent($childDest, $childSource);
+        $this->eventDispatcher->dispatch(ContentEvents::NODE_ORDER_BEFORE, $event);
```

Comment:
ah ok i see the orderBefore event is fired multiple time when i execute oder before ... i think thats not a good idea if someone uses this event ... perhaps we should provide an `order` event and pass there the needed information ... this is a bc break but for me its ok /cc @danrot